tests/int/test-git_mirror.bsh

```diff
@@ -686,14 +612,11 @@ begin_test "Part 7 - Rewrite a ref and mirror"
   popd &> /dev/null
 
   # Push to mirror
-  # NOTE These regex's are matching long-form refs
-  GIT_MIRROR_TAG_REFS_REGEX='.+to_tag_.+' \
-      GIT_MIRROR_PUSH_REFS_REGEX='.+_push.+' \
-      GIT_MIRROR_SKIP_REFS_REGEX='refs/heads/to_tag_skip_.+' \
+  # NOTE These regex's are matching long-form refs (this regex is using a
+  # negative look-ahead)
+  GIT_MIRROR_TAG_REFS_REGEX='^(?!no_).+$' \
```

Comment:
Bash doesn't support look ahead

```
$ x="yolo yololo"
$ p='(?!lo)'
$ echo 'yolo yololo' | perl -wpe 's/yolo(?!lo)/YOLO/'
YOLO yololo
$ [[ ${x} =~ ${p} ]]
$ declare -p | grep BASH_RE
13:declare -ar BASH_REMATCH=()
```

Reply:
So it would seem. It is odd that the tests all happily passed